roles/ceph-monitor/tasks/main.yml

```diff
@@ -55,6 +55,20 @@
 - name: wait for client.admin key exists
   wait_for: path=/etc/ceph/ceph.client.admin.keyring
 
+- name: make sure ceph.log has proper permissions for logstash
```

Comment:
these files are created and logrotated and thus trying to manage their perms directly in ansible is a losing battle.   We should ensure the directory housing them has the correct perms for this to inherit.